public/scss/main.scss

```diff
@@ -980,6 +980,9 @@ $letter-spacing: 0px;
 		p{
 			color: #333;
 		}
+		.team-bio {
+			font-size: 12px;
```

Comment:
Does the stylesheet use `px`? Does this project use responsive units anywhere? (`rem` is usually preferred but depends on project. If entire project uses px then stick with `px`).

Reply:
Yuuuuuup